Makefile

```diff
@@ -37,7 +37,9 @@ OBJS = $(patsubst %.riscv, %.o,$(EHOST)) $(KEYSTONE_OBJ) edge_wrapper.o
 
 TCLIENT_OBJS = $(patsubst %.cpp, %.o,$(TCLIENT))
 
-all:  $(OBJS) $(SDK_HOST_LIB) $(SDK_EDGE_LIB) $(SDK_VERIFIER_LIB) $(SODC_LIB)
+all: makeall copysdk getandsethash trusted_client.riscv copysdk1
```

Comment:
Rather than do the copysdk1 thing, can we split this up so that `all` does the full build, and we have another target that does the gethash thing and rebuild? Not everyone building the demo will be targeting qemu, and may not even have the qemu build setup.

ex:
```
all: $(OBJS) $(SDK_HOST_LIB) $(SDK_EDGE_LIB) $(SDK_VERIFIER_LIB) $(SODC_LIB) trusted_client.riscv

Reply:
I see what you mean, I will make it a separate target which will do this so that all will just do what it did previously.